src/services/projects/projectsTypes.ts

```diff
@@ -0,0 +1,19 @@
+// To-do add generic response type with a status attr in a general types file?
+
+export interface Project {
```

Comment:
This type could be imported from the src/types folder to avoid doubly defining it.